CRM/Mailing/BAO/Mailing.php

```diff
@@ -1470,7 +1470,7 @@ public static function add(&$params, $ids = []) {
    * @throws \CRM_Core_Exception
    * @throws \CiviCRM_API3_Exception
    */
-  public static function create(&$params) {
+  public static function create(array $params) {
```

Comment:
So this seems to work @eileenmcnaughton but some recent experiences had made me wary of retroactively adding type hints to old functions. It has a tendency to make things blow up and I don't really see the benefit.